storage/modules/balance_storage.go

```diff
@@ -187,6 +187,10 @@ func (b *BalanceStorage) AddingBlock(
 	block *types.Block,
 	transaction database.Transaction,
 ) (database.CommitWorker, error) {
+	if b.handler == nil {
```

Comment:
Can we force the handler to be present in the methods used to create a new BalanceStorage instance?

Reply:
I wish! There is a somewhat complex dependency process where we can't add the handler or helper during initialization.

Reply:
We need a reference to `BalanceStorage` to create some useful helpers (we should probably clean this up): https://github.com/coinbase/rosetta-cli/blob/972a1690001f7f7d22fafa0a580512eac8517add/pkg/tester/data.go#L284-L305

Reply:
yeah, we should def clean it up!